build.gradle

```diff
@@ -104,6 +104,7 @@ dependencies {
 
     compile 'org.jsoup:jsoup:1.9.1'
     compile 'com.mashape.unirest:unirest-java:1.4.9'
+    compile 'info.debatty:java-string-similarity:0.13'
```

Comment:
must be added to https://github.com/JabRef/jabref/blob/master/external-libraries.txt